client/extensions/wp-job-manager/state/data-layer/settings/utils.js

```diff
@@ -0,0 +1,37 @@
+export const fromApi = data => ( {
+	listings: {
+		perPage: data.job_manager_per_page,
+		hideFilledPositions: data.job_manager_hide_filled_positions,
+		hideExpired: data.job_manager_hide_expired,
+		hideExpiredContent: data.job_manager_hide_expired_content,
+	},
+	categories: {
+		enableCategories: data.job_manager_enable_categories,
```

Comment:
not sure if we're limited by other names, but we have an inherent namespacing here by means of the nesting. I wonder if it's enough to have `categories.isEnabled` and `types.isEnabled` instead of `categories.enableCategories` and `types.enableTypes` etc…